Khan/perseus
# [Interactive Graph Editor] Add/Edit/Delete Locked Function

## Summary:
Content editors need a way to add and edit a locked function. This PR adds a settings section that is specific to functions. 

Issue: LEMS-1947

## Test plan:
1. Launch Storybook (yarn start)
1. Navigate to Perseus Editor => Widgets => Interactive Graph Editor => [With Locked Lines](http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph-editor--with-locked-lines)
1. Click on the "Add locked figure" drop-down at the bottom of the editor and choose "Function"
   * A function with default settings should be added to the example graph
   * A settings section should be added to the bottom of the list of items on the graph
   * The settings should be open, and should show the color, stroke, function equation, and domain settings
1. Changing any of the settings should immediately update the function in the example graph

## Affected UI
### New settings options
![Locked Function Settings UI](https://github.com/user-attachments/assets/04770c83-cbc8-4b72-b495-ba4bcd92162a)
